### PR TITLE
create-manager

### PR DESCRIPTION
**Pull request by Bayka1997**
==> Title : Create manager

==> Redmine:
https://edu-redmine.sun-asterisk.vn/issues/16519
==> Attachment File:

### CODE REVIEW

**Comment by Bayka1997** [comment]
ready

File: app/assets/javascripts/admin/custom.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    dataType: 'script',
+    data: {'id' : id}
+  });
+}
+
+function reset_errors(type) {
+  var field = ['name', 'email', 'address',
+    'phone', 'password', 'password_confirmation'];
+  $.each(field, function (value) { 
+      $('#error_' + value + type).html('');
```
Comment (hungct-1121) [inline app/assets/javascripts/admin/custom.js]:
```suggestion
    $('#error_' + value + type).html('');
```

File: app/assets/javascripts/admin/custom.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+  $.ajax({
+    type: form.attr('method'),
+    url: form.attr('action'),
+    data: form.serialize(),
+    success: function (data) {
+      reset_errors('_new');
+      if(data.errors) {
+        show_errors(data.errors, '_new');
+      }else {
```
Comment (hungct-1121) [inline app/assets/javascripts/admin/custom.js]:
```suggestion
      } else {
```

**Comment by Bayka1997** [comment]
ready

File: app/controllers/admin/users_controller.rb [hunk 3]
```diff
@@ -1,11 +1,40 @@
...
+  before_action :load_user, except: %i(index create new_user)
 
   def index
     @search = User.ransack params[:search]
     @users = @search.result.page(params[:page]).per Settings.admin.user.num_in_page
   end
 
+  def new_user
+    @user = User.new
+
```
Comment (hungct-1121) [inline app/controllers/admin/users_controller.rb]: ko cần thêm dòng trống

**Pull request merged**